### PR TITLE
New negative trait: Brain Tumor

### PR DESCRIPTION
**Pull request by ghost**
I like the new trait system let's add some traits to it

### CODE REVIEW

**Comment by Armhulen** [comment]
It slowly adds brain damage until death, better bring mannitol!

**Comment by Repukan** [comment]
Wouldn't it be better to add a bunch instead of one by one

**Comment by KorPhaeron** [comment]
>get brain tumour trait
>go get it removed rounstart
>free trait points

?

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -13,6 +13,17 @@
 
 
 
+/datum/trait/morerobustthanyou
+	name = "Brain Tumor"
+	desc = "You got a little friend in yo head thats drivin you insane in the membrane."
+	value = -2
+	mob_trait = TRAIT_HEAVY_SLEEPER
```
Comment (XDTM) [inline code/datums/traits/negative.dm]: .

Comment (Armhulen) [reply code/datums/traits/negative.dm]: Uuuuuum whoops I'll fix that

**Comment by Qustinnus** [comment]
should also ban you from using goonchat just like @MoreRobustThanYou

File: code/datums/traits/negative.dm [hunk 2]
```diff
@@ -13,6 +13,17 @@
...
+	gain_text = "<span class='danger'>You feel smooth.</span>"
+	lose_text = "<span class='notice'>You feel wrinkled again.</span>"
+
+/datum/trait/morerobustthanyou/on_process()
+	adjustBrainLoss(1)
```
Comment (Qustinnus) [inline code/datums/traits/negative.dm]: stop calling procs on the wrong datums you chumbis

**Comment by Armhulen** [comment]
Don't be on the nose

File: code/datums/traits/negative.dm (hunk 2 above)
Comment (Armhulen) [reply code/datums/traits/negative.dm]: Oh lol

**Comment by Tlaltecuhtli** [comment]
make it an admin verb

**Comment by Spaceinaba** [comment]
you can now roleplay as the average irl ss13 player

**Comment by Cyberboss** [comment]
That's just mean

**Comment by Repukan** [comment]
24 hr rule?

Mods?

**Comment by PKPenguin321** [comment]
its because the code has player reference

**Comment by PKPenguin321** [comment]
here arm no more player reference: https://github.com/Armhulen/tgstation/pull/9

**Comment by Qustinnus** [comment]
@PKPenguin321 That's dumb because the player reference is part of the code, not part of the ingame experience. It's as much a player reference as a comment saying //This is for MRTY.

**Comment by PKPenguin321** [comment]
i think its dumb too but it got the pr closed

**Comment by Armhulen** [comment]
I'll do it with no player reference FINE

**Pull request closed without merging**